src/NuGet.Core/NuGet.Frameworks/NuGetFrameworkFactory.cs

```diff
@@ -650,6 +650,12 @@ private static bool TryParseCommonFramework(string frameworkString, [NotNullWhen
                 case "net70":
                     framework = FrameworkConstants.CommonFrameworks.Net70;
                     break;
+                case "netcoreapp8.0":
+                case "netcoreapp80":
+                case "net8.0":
+                case "net80":
+                    framework = FrameworkConstants.CommonFrameworks.Net80;
+                    break;
```

Comment:
When I do something that has been done before --- like adding a new TFM ---, I review how others did it in the past.  Not to copy, but to see if there's something I may have missed.

[This](https://github.com/NuGet/NuGet.Client/commit/820bfe9d5e5e4e7bf0b6310c285db7b07aff3087#diff-9608c8cf5369cc492f2ed8d56c58057607817e6a6e1eb776854f24effc46e4b1R434) looks like a good place to add a test for `net8.0`.  Bonus points for adding `net7.0` since its PR overlooked it too.

Even without this change, restore works for `net8.0`.  The gist of this change is that we now recognize `net8.0` (and related TFM's) and return a static instance.  A good rule of thumb is that if it's worth making a product change, it's probably also worth adding a test.

Don't forget to update the PR description to indicate that tests were added.

Reply:
That makes sense. I've now added tests covering the changes, and updated the PR description too